gradio/components/multimodal_textbox.py

```diff
@@ -0,0 +1,196 @@
+"""gr.MultimodalTextbox() component."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Literal
+
+import gradio_client.utils as client_utils
+from gradio_client.documentation import document
+
+from gradio import processing_utils
+from gradio.components.base import FormComponent
+from gradio.data_classes import FileData, GradioRootModel
+from gradio.events import Events
+
+
+class MultimodalData(GradioRootModel):
```

Comment:
I think it would be better to name the fields explicitly

```python
from pydantic import Field

class MultimodalData(GradioModel):
    text: Optional[str] = None
    files: Optional[List[FileData]] = Field(default_factory=list)
```